src/test/java/com/fittracker/fittracker/entity/ProductTest.java

```diff
@@ -0,0 +1,40 @@
+package com.fittracker.fittracker.entity;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.time.LocalDateTime;
+import java.util.UUID;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+
+class ProductTest {
+
+    private static final UUID TEST_UUID = UUID.fromString("084ce4d3-fbc2-45ff-9425-2f2ce116c512");
+
+    private static final UUID ANOTHER_TEST_UUID = UUID.fromString("eabd0b51-87c4-40f2-a0a7-f242215275e2");
+    private static final UUID TEST_USER_UUID = UUID.fromString("a425d08e-f37b-48d9-8acc-9580cd574ffd");
+
+    private static final LocalDateTime TEST_TIMESTAMP = LocalDateTime.of(2023, 10, 10, 4, 20);
+
+    @Nested
+    class HasEqualData {
```

Comment:
Very good point!

On a side note: we are starting to get a lot of static methods in our `dataprovider`. I was thinking about substituting sets of methods related to given entity with one method using reflection to modify field. For `Product` it would look a little bit like this:

```
public static Product createProductWithField(String fieldName, Object value) {
        Product product = Product.product();

        try {
            Field field = Product.class.getDeclaredField(fieldName);
            field.setAccessible(true); // allows access to private fields
            field.set(product, value);
        } catch (NoSuchFieldException | IllegalAccessException e) {
            e.printStackTrace(); // handle exceptions as appropriate
        }

        return product;
}
```
What do you think about this? Maybe we should talk about it too?

Reply:
as discussed, it's ok for now, especially considering it's test code, resolving